### PR TITLE
add left time in progress hook

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: chainercv/utils/iterator/progress_hook.py [hunk 1]
```diff
@@ -23,9 +23,12 @@ def __call__(self, in_values, out_values, rest_values):
         self.n_processed += len(in_values[0])
         fps = self.n_processed / (time.time() - self.start)
         if self.n_total is not None:
+            eta = int((self.n_total - self.n_processed) / fps)
             sys.stdout.write(
-                '\r{:d} of {:d} samples, {:.2f} samples/sec'.format(
-                    self.n_processed, self.n_total, fps))
+                '\r{:d} of {:d} samples, {:.2f} samples/sec,'
+                ' {:4d}:{:02d}:{:02d}'.format(
```
Comment (Hakuyume) [inline chainercv/utils/iterator/progress_hook.py]: How about `ETA {:4d}:{:02d}:{:02d` ?

**Comment by yuyu2172** [comment]
Can you check the case when `fps=0` (This could technically happen when  `len(in_values[0])`)?

File: chainercv/utils/iterator/progress_hook.py (hunk 1 above)
Comment (knorth55) [reply chainercv/utils/iterator/progress_hook.py]: updated

**Comment by knorth55** [comment]
updated

**Comment by yuyu2172** [comment]
LGTM!

**Pull request merged**